trello_cards.py

```diff
@@ -0,0 +1,72 @@
+import requests
+from config import api_key, api_token
```

Comment:
If you want a secret file, you should add it to the .gitignore file, and document it in the README.md.

But ideally you should commit all python files, and you can load secrets from environment variables. e.g. `trello_api_key = os.getenv('TRELLO_API_KEY')`. Then you just need to add `TRELLO_API_KEY=yourkey` etc to .env / .env.template and document it in the README